README.md

```diff
@@ -42,16 +41,15 @@ Ruby setup, you might need to run `bundle exec citizen` instead of just `citizen
 
 ### Deploying
 
-* [CircleCI](https://circleci.com/gh/codeforamerica/michigan-benefits) is currently set up to 
+* [CircleCI](https://circleci.com/gh/codeforamerica/michigan-benefits) is currently set up to
```

Comment:
I was not able to figure out the badge stuff before when I was setting up something for code climate. I was trying to add them for Circle and Code Climate (test coverage and rating) and 1 of them was giving me issue. Added a small cleanup task here: https://trello.com/c/JPeLG4LX